package ExceptionHandling;

public class ExceptionTest {
    String name = "Java";
    void test(){
        System.out.println("Test method called");
    }

    public static void main(String[] args) {
        System.out.println("Main method execution started");
       /* ExceptionTest test = null;
        test.test();
        try {
            ExceptionTest test1 =null;
            test.test();
        }catch (Exception e){

        }*/
        System.out.println("Test line execution");
    }
}
